src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.figure;
+
+import base.Figure;
+
+public class IsoscelesTrapezoid extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line above the class declaration.